hub/powertoys/run.md

```diff
@@ -117,7 +111,7 @@ The Windows System Commands plugin provides a set of system level actions that c
 | `Sleep` | Puts the computer to sleep | |
 | `Hibernate` | Hibernates the computer | |
 | `Empty Recycle Bin` | Empties the recycle bin | |
-| `UEFI Firmware Settings` | Reboots the computer into UEFI Firmware Settings | Only available on systems with UEFI firmware.<br />(Requires administrative permissions.) |
+| `UEFI Firmware Settings` | Reboots the computer into UEFI Firmware Settings | Only available on systems with UEFI firmware. Requires administrative permissions. |
```

Comment:
```suggestion
| `UEFI Firmware Settings` | Reboots the computer into UEFI Firmware Settings | Only available on systems with UEFI firmware. (Requires administrative permissions.) |
```

Reply:
Why are these so important?

Reply:
I think this is a side note wich note. The function imo is more important. So the brakets split function description and note.

Reply:
Ooh. Sory. Didn't saw that it is in the last row. So it ojay as is